pkg/operator/sync.go

```diff
@@ -126,6 +146,65 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	err := optr.syncValidatingWebhook(optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(), newValidatingWebhook(config))
+	if err != nil {
+		return err
+	}
+	err = optr.syncMutatingWebhook(optr.kubeClient.AdmissionregistrationV1().MutatingWebhookConfigurations(), newMutatingWebhook(config))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (optr *Operator) syncValidatingWebhook(client admissionclientv1.ValidatingWebhookConfigurationInterface, hook *admissionregistrationv1.ValidatingWebhookConfiguration) error {
+	validHook, err := optr.validatingWebhookLister.Get(hook.Name)
+	if apierrors.IsNotFound(err) {
+		_, err = client.Create(context.Background(), hook, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		return nil
+	} else if err != nil {
+		return err
+	}
+	if !equality.Semantic.DeepDerivative(hook.Webhooks, validHook.Webhooks) {
```

Comment:
Maybe we update this with a unstructured apply at some point, but for now I think it's ok if you're happy with it and have tested it @Danil-Grigorev

Reply:
Why are we using `DeepDerivative` here?

Reply:
hook vs validHook seems confusing names. Can we may be use  expected vs current/got

Reply:
Please remember the `caBundle` field, and the fact that `c.Get` on that resource will never return fully equal resource as the one we are trying to apply. The `deepDerivative` on hooks tells for sure, if some of the webhooks was added or changed - our goal of management with operator.

Reply:
yeh that's clear to me. But it won't be in month from now or for anyone looking at this code with less context coming from somewhere else.
Can we please add a comment explaining the reasoning for this choice?

Reply:
This is still seems to not be addressed? Just a comment on top of 175 would be very valuable.

Reply:
I accidentally skipped this during one rebase.